USER_GUIDE.md

```diff
@@ -53,6 +53,24 @@ If `comments` is given in the `features` list then this enables all commenting f
 
 > Note: All commands can be given with a prefix of either `Derek <command>` or `/<command>`.
 
+### Feature: `messages`
+
+Boilerplate messages can be applied for the time you would like to point someone to the contributing guide for example. You can just write `Derek message: docs` or `/message: docs` you can also use the short style `msg` instead of `message`. Then the content of that message is displayed.
+
+Configure the feature in `.DEREK.yml` file. It should look something like:
+
+```
+messages:
+  - name: docs
+    value: Hello, please check out the docs ...
+  - name: slack
+    value: |
+           -- 
+           To join our slack channel ...
+```
+
+Above are two examples which shows simple configuration, the first one is the normal way, the second one is more specific multi line literal, which should be exactly below the `|` sign in order to be displayed and not having errors while parsing.
```

Comment:
Normal?  Maybe it's the method for single line messages.

Reply:
Swapped `the first one is the normal way` -> `the first one is the method for single line messages`